metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
In Packetbeat we have [`responsetime`](https://www.elastic.co/guide/en/beats/packetbeat/1.1/exported-fields-measurements.html#_responsetime) and it is measured in milliseconds. This will create an inconsistency.